lib/kele.rb

```diff
@@ -15,13 +15,37 @@ def initialize(email, pass)
   end
 
   def get_me
-    response = self.class.get('/users/me', headers: { authorization: @auth_token })
+    response = self.class.get('/users/me', headers: auth_header)
     JSON.parse(response.body)
   end
 
   def get_mentor_availability(mentor_id)
-    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: { authorization: @auth_token})
+    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: auth_header)
     JSON.parse(response.body)
   end
 
+  def get_messages(page_number = nil)
+    if page_number != nil
+      @page = {page: page_number}
+      response = self.class.get('/message_threads', headers: auth_header, body: @page)
+    else
+      response = self.class.get('/message_threads', headers: auth_header)
+    end
+    JSON.parse(response.body)
+  end
+
+  def create_message(sender, recipient, token, subject, text)
+    @body = {"sender" => sender, "recipient_id" => recipient, "token" => token, "subject" => subject, "stripped-text" => text}
+    if @body["token"] == nil
```

Comment:
This looks good! But can be cleaned up like this `@body.delete('token') if @body['token'].nil?`